include/solvers/navier_stokes_scratch_data.h

```diff
@@ -447,6 +457,16 @@ class NavierStokesScratchData
              const Quadrature<dim> &   quadrature,
              const Mapping<dim> &      mapping);
 
+  void
+  enable_pfg(const FiniteElement<dim> &fe_pfg,
```

Comment:
Same to replace pfg with phase_filtered_gradient (even add a vof in front)

Reply:
done.